shared/scss/abstracts/_media-query.scss

```diff
@@ -15,8 +15,6 @@
 	// @import "compass/css3";
 	// @import "compass/utilities/general/clearfix";
 
```

Comment:
remove comments if you don't want to use it

Reply:
OK